docs/userguide/networking/work-with-networks.md

```diff
@@ -0,0 +1,454 @@
+
+* `docker network create`
+* `docker network connect`
+* `docker network ls`
+* `docker network rm`
+* `docker network disconnect`
+* `docker network inspect`
+
+While not required, it is a good idea to read [Understanding Docker
+network](dockernetworks.md) before trying the examples in this section. The
+examples for the rely on a `bridge` network so that you can try them
+immediately.  If you would prefer to experiment with an `overlay` network see
+the [Getting started with multi-host networks](get-started-overlay.md) instead.
+
+## Create networks
+
+When you install Docker Engine it creates a `bridge` network automatically.
+This network corresponds to the `docker0` bridge that Engine has traditionally
+relied on. In addition to this network, you can create your own `bridge` or `overlay` network.  
+
+A `bridge` network resides on a single host running an instance of Docker Engine.  An `overlay` network can span multiple hosts running their own engines. If you run `docker network creat`e and supply only a network name, it creates a bridge network for you.
+
+```bash
+$ docker network create simple-network
+de792b8258895cf5dc3b43835e9d61a9803500b991654dacb1f4f0546b1c88f8
+$ docker network inspect simple-network
+[
+    {
+        "Name": "simple-network",
+        "Id": "de792b8258895cf5dc3b43835e9d61a9803500b991654dacb1f4f0546b1c88f8",
+        "Scope": "local",
+        "Driver": "bridge",
+        "IPAM": {
+            "Driver": "default",
+            "Config": [
+                {}
+            ]
+        },
+        "Containers": {},
+        "Options": {}
+    }
+]
+```
+
+Unlike `bridge` networks `overlay` networks require some pre-existing conditions
+before you can create one. These conditions are:
+
+* Access to a key-value store. Engine supports Consul Etcd, and Zookeeper (Distributed store) key-value stores.
+* A cluster of hosts with connectivity to the key-value store.
+* A properly configured Engine `daemon` on each host in the cluster.
+
+The `docker daemon` options that support the `overlay` network are:
+
+* `--cluster-store`
+* `--cluster-store-opt`
+* `--cluster-advertise`
+
+It is also a good idea, though not required, that you install Docker Swarm on
+to manage the cluster. Swarm provides sophisticated discovery and server
+management that can assist your implementation.
+
+When you create a network, Engine creates a non-overlapping subnetwork for the network by default. You can override this default and specify a subnetwork directly using the the `--subnet` option. On a `bridge` network you can only create a single subnet. An `overlay` network supports multiple subnets.
+
+In addition to the `--subnetwork` option, you also specify the `--gateway` `--ip-range` and `--aux-address` options.
+
+```bash
+docker network create -d overlay
```

Comment:
Missing `$` prompt. Also, consider wrapping options to have one option/flag per line